src/main/java/com/jmb/springfactory/model/entity/ProductionSchedule.java

```diff
@@ -0,0 +1,38 @@
+package com.jmb.springfactory.model.entity;
+
+import java.util.Date;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import com.jmb.springfactory.model.enumeration.ProductionScheduleStateEnum;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+@NoArgsConstructor
+@Entity
+@Table
+public class ProductionSchedule extends BaseEntity {
+
+  private String name;
+  
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date estimatedStartDate;
+  
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date estimatedFinishDate;
+  
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date realStartDate;
+  
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date realFinishDate;
+  
+  private ProductionScheduleStateEnum state;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "state" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)